src/graphics/program-lib/chunks/ambientPrefilteredCube.frag

```diff
@@ -4,7 +4,9 @@ uniform samplerCube texture_prefilteredCubeMap4;
 #endif
 
 void addAmbient() {
-    vec3 fixedReflDir = fixSeamsStatic(dNormalW, 1.0 - 1.0 / 4.0);
+    vec3 fixedReflDir = fixSeamsStatic(cubeMapRotate(dNormalW), 1.0 - 1.0 / 4.0);
+#ifndef DYNCUBEMAP
```

Comment:
Wouldn't it be clearer to readers if this macro was named such that it describes better what it does? Perhaps REFLECT_CUBE_X or similar?

Reply:
I thought about this - I would prefer the macro name to give an idea what it is used for - not what it does. Its used to specify if the cubemap is dynamic and when the cubemap is not dynamic it skips an operation

Reply:
hmm - how about RIGHT_HANDED_CUBEMAP - this is the real difference between dynamically rendered cubemaps and the pregenerated cubemaps - and what this macro and the flip operation deals with?

Reply:
yes!!!

Reply:
I like that!

Reply:
renamed macros - thanks!